src/js/schemas/job-schema/Schedule.js

```diff
@@ -35,7 +35,11 @@ const General = {
     },
     cron: {
       title: 'CRON Schedule',
-      description: 'Enter your schedule in CRON format, e.g. 0 20 * * *',
+      description: (
+        <span>
+          Enter your schedule in CRON format, e.g. <i>0 0 20 * * *</i>. <a href="https://dcos.io/docs/1.8/usage/jobs/getting-started" target="_blank">Read the docs</a> for more info.
```

Comment:
@pierlo-upitup @wavesoft: FYI: We need to make sure to not include links with versions in them, we have `/latest` for that, which redirects you to the latest docs. Also, we have `MetadataStore.buildDocsURI` function that will make sure to distinguish open vs. enterprise